pkg/config/config.go

```diff
@@ -18,10 +17,34 @@ var (
 		"instance-url",
 		field.WithDescription(`The GitHub instance URL to connect to. (default "https://github.com")`),
 	)
+	appIDField = field.StringField(
+		"app-id",
+		field.WithDescription("The GitHub App to connect to."),
+	)
+
+	appPrivateKeyPath = field.StringField(
+		"app-privatekey-path",
+		field.WithDescription("Path to private key that is used to connect to the GitHub App"),
+	)
 	syncSecrets = field.BoolField(
 		"sync-secrets",
 		field.WithDescription(`Whether to sync secrets or not`),
 	)
+	installationID = field.StringField(
```

Comment:
can we not use the org to get this? not sure if customers have this ID on hand 

https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-an-organization-installation-for-the-authenticated-app